tracing-subscriber/src/registry/stack.rs

```diff
@@ -58,3 +64,27 @@ impl SpanStack {
 thread_local! {
     static CONTEXT: RefCell<SpanStack> = RefCell::new(SpanStack::new());
 }
+
+#[cfg(test)]
+pub(crate) mod tests {
```

Comment:
It's a little weird to see a `pub(crate)` on the test module. not a blocker though.

Reply:
I was curious about that too. I was using the tests in `Registry` as an example which also specifies `pub(crate)`. Will remove.

Reply:
Ah, I _think_ the `Registry` tests once had some test-utility code that was used by tests in another module, and that the code was removed but we forgot to make the module private again? Just a guess.